### PR TITLE
master.toml: Cache mafia builds

### PR DESCRIPTION
**Pull request by erikd-ambiata**
! @jystic @tmcgilchrist

### CODE REVIEW

File: master.toml [hunk 1]
```diff
@@ -5,40 +5,46 @@
...
 [build.dist-8-0]
   GHC_VERSION="8.0.1"
   CABAL_VERSION="1.24.0.0"
   MAFIA_HAPPY = "true"
+  CACHE = "true"
 
 [build.branches-7-8]
   GHC_VERSION = "7.8.4"
-  CABAL_VERSION = "1.22.4.0"
+  CABAL_VERSION = "1.24.0.0"
```
Comment (tmcgilchrist) [inline master.toml:31]: Does that version of cabal work with GHC 7.8.4?

**Comment by tmcgilchrist** [comment]
If it builds sure 👍

File: master.toml (hunk 1 above)
Comment (erikd-ambiata) [reply master.toml:31]: I though it does. We'll see soon enough. If not, I'll remove caching for the 7.8 build.

**Comment by erikd-ambiata** [comment]
Mafia builds in CI using a `cabal` script that is stored in the repo. Not sure if caching makes sense. 

@jystic ?

File: master.toml (hunk 1 above)
Comment (jacobstanley) [reply master.toml:31]: caching doesn't work for 7.8 anyway

**Comment by jacobstanley** [comment]
> Mafia builds in CI using a cabal script that is stored in the repo. Not sure if caching makes sense.

I think actually mafia is built using mafia because of the master runner which is in use.

**Comment by jacobstanley** [comment]
Ah, having said that, the master runner it's using doesn't support caching as it is too old

**Comment by jacobstanley** [comment]
I think you might potentially run in to a chicken and egg problem if you upgrade the master runner, not sure

**Comment by erikd-ambiata** [comment]
Updated the master runner to see if it works.

**Pull request merged**